src/Controller/IndexController.php

```diff
@@ -546,6 +546,17 @@ public function model_import(Request $request, Session $session)
             $dates[0] = $date;
         }
 
+        // Remove locks from associated plannings
```

Comment:
Technically correct, but for new code like this, shouldn't we be doing it the Symfony way?

see src/PlanningBiblio/DataPurger.php for PlanningPositionLock deletion based on dates (simplePurge function)

I would recommand creating an src/Repository/PlanningPositionLockRepository.php and add a function there to take care of the deletion.

Reply:
done: 827973e57c7fe3a5b3c6007b47e27883bf101161